trunk_lexer/Cargo.toml

```diff
@@ -4,4 +4,7 @@ version = "0.1.0"
 edition = "2021"
 
 [lib]
-doctest = false
\ No newline at end of file
+doctest = false
+
+[dependencies]
+serde = { version = "1.0.139", features = ["derive"] }
```

Comment:
The lexer crate now has to depend on serde, since `ByteString` needs a `Serialize` impl so that all the AST types can derive an impl.

Reply:
I'm happy to remove `serde` right now actually - I don't need to serialize the AST at all and it was originally for some meta scripts for debugging.

Reply:
Sweet, done 👍